src/main/java/com/faforever/api/data/domain/Clan.java

```diff
@@ -47,6 +47,7 @@
   private String description;
   private String tagColor;
   private String websiteUrl;
+  private Boolean requiresInvitation = Boolean.TRUE;
```

Comment:
Is that `Boolean` because of database definitions?

Reply:
needs to be Boolean instead of boolean if it is nullable